tools/topology/sof-tgl-max98373-rt5682.m4

```diff
@@ -92,12 +92,25 @@ dnl PIPELINE_PCM_ADD(pipeline,
 dnl     pipe id, pcm, max channels, format,
 dnl     frames, deadline, priority, core)
 
+ifelse(INCLUDE_IIR_EQ, `1',
```

Comment:
i think ifdef would be simpler here

Reply:
@cujomalainey Forgot if that thing actually works in m4.

Reply:
I guess it depends on whether it's likely more pipelines are going to be added to this topology. If that's likely then perhaps ifelse is better to allow them to be added otherwise I can change it ifdef

Reply:
@RDharageswari @sathya-nujella can you comment on the last comment. It's an easy change to make later if needed.

Reply:
@rosschisholm-xperi can you make a quick chnage here to ```ifdef``` as this may be the single user atm for this topology. We can alway update it later when more users are added. Thanks.

Reply:
Change made to ifdef